website/docs/docs/authentication/legacy.md

```diff
@@ -0,0 +1,490 @@
+
+## SAML Support
+
+_Eligible plans: Enterprise_
+
+PactFlow supports [SAML 2.0](https://en.wikipedia.org/wiki/SAML_2.0) integration with a compatible Identity Provider (IdP) such as Okta, Ping, Auth0, Azure AD, OneLogin etc.
+
+SAML2.0 allows you to externalise the authentication and access to your PactFlow account. Authorization and fine-grained permissions are managed within your account by an account Administrator.
+
+Supported capabilities:
+
+- Authorization from an external IdP
+- Automated user provisioning into your account
+- SP initiated login
+- [SCIM](/docs/scim/main)
+
+We do not currently support the following:
+
+- Automated user deprovisioning (users will appear "active" and count toward user limits, although will not be able to login if disabled in the IdP)
+- IdP initiated login
+- Service Provider (SP) initiated logout flow
+
+### Setup
+
+#### 1. Configure PactFlow as a Service Provider in your IdP
+
+In your IdP, create a new Service Provider with the following properties:
+
+- Audience URI (SP Entity ID): `urn:amazon:cognito:sp:ap-southeast-2_x0L1olP0D`
+- Single sign on URL (Reply URL): `https://pact-saas-prod-1.auth.ap-southeast-2.amazoncognito.com/saml2/idpresponse`
+- Name ID format must be set to "Persistent": `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`
+
+#### 2. Map the required SAML2.0 attributes
+
+The following attributes are required by PactFlow and must be mapped in your IdP to be sent through during the authentication flow:
+
+| Property to map from your IDP | Attribute Name in SAML Assertion                                     | Name Format   |
+| ----------------------------- | -------------------------------------------------------------------- | ------------- |
+| First Name                    | `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`    | URI Reference |
+| Last Name                     | `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname`      | URI Reference |
+| Email Address                 | `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress` | URI Reference |
+
+For example, a valid Attribute in the SAML assertion for a user's first name would look like this:
+
+```
+<Attribute Name="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri">
+  <AttributeValue xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="xs:string">Joe</AttributeValue>
+</Attribute>
+```
+
+#### 3. Export IdP metadata
+
+Once you have configured PactFlow as a Service Provider, please send your PactFlow account details and an externally accessible URL to the metadata file via our [support form](https://support.smartbear.com/pactflow/message/). If it is not possible to provide a URL, the metadata may be exported as an XML file. It is preferable to use a URL, as this will allow you to make any updates without contacting PactFlow support.
```

Comment:
If it is impossible to provide a URL, you may export the metadata as an XML file.